internal/backends/postgresql/collection.go

```diff
@@ -321,12 +322,13 @@ func (c *collection) Explain(ctx context.Context, params *backends.ExplainParams
 
 	q += where
 
-	sort, sortArgs := prepareOrderByClause(&placeholder, params.Sort, meta.Capped())
+	sort, sortArgs, sortPushdownAll := prepareOrderByClause(&placeholder, params.Sort, meta.Capped())
```

Comment:
In handler code, we don't set `params.Sort` if there is more than one field for sort, for example `{v: 1, _id: 1}`. So `params.Sort` is nil that case.
https://github.com/FerretDB/FerretDB/blob/0e69b3dfed7d18daafb0d7e2b1de313ed41af4e4/internal/handlers/sqlite/msg_explain.go#L91

`prepareOrderByClause()` returns boolean `true` seeing `params.Sort == nil`, which then allows `LIMIT` to be applied. This will be an issue because `LIMIT` would be applied before sorting documents in memory.

Let's be cautious and ensure we have tests to cover this case. :hugs:

Reply:
handler code is updated. And a test case with two fields for sort is added to cover this case.